qemu/tests/hotplug_virtio_mem.py

```diff
@@ -26,6 +35,10 @@ def run(test, params, env):
     timeout = params.get_numeric("login_timeout", 240)
     threshold = params.get_numeric("threshold", target_type=float)
     error_msg = params.get("error_msg")
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = utils_qemu.get_qemu_version(qemu_path)
+    match = re.search(r"[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?", str(qemu_version))
+    host_qemu = match.group(0)
```

Comment:
```suggestion
    qemu_path = utils_misc.get_qemu_binary(params)
    qemu_version = utils_qemu.get_qemu_version(qemu_path)[0]
```

HIi @mcasquer 
I think that we could get the `host_qemu`(qemu_version) by `qemu_version[0]` directly.